### ISSUE TITLE
Fast tokenizer and chunksize

### ISSUE THREAD

**Issue by DelinQu** [declare-lab/nora#5]
Hi [hungchiayu1 Chia-Yu Hung](https://github.com/hungchiayu1), 
Thank you for open-sourcing your work. I'm currently using Qwen2.5 VL with the fast tokenizer to generate results, but I'm encountering some unusual outputs even when following your recommended training settings (chunksize=4). This issue might be related to the inherent limitations of the fast tokenizer.

I'd like to ask how NoRA managed to avoid this problem and maintain excellent performance. Did your team retrain or modify the fast tokenizer in the bridge and fractal datasets to address these issues? I'd appreciate any insights you could share about your approach to handling tokenization challenges while achieving such impressive results.

![Image](https://github.com/user-attachments/assets/36edf15d-d170-4e86-9d15-9851bf578fd2)

**Comment by soujanyaporia** [comment]
I will let @hungchiayu1 to assist. But FYI we have released all the checkpoints fine-tuned on Libero. Please feel free to use those: https://huggingface.co/collections/declare-lab/nora-6811ba3e820ef362d9eca281

**Comment by DelinQu** [comment]
Thanks [soujanyaporia](https://github.com/soujanyaporia), 

I noticed that in NORA, the action_dim is set to 7 ([code reference](https://github.com/declare-lab/nora/blob/4d74bf2ab7db2ced13cc8cb013a206ebfc7a716c/inference/nora.py#L6)), unlike the original pi0-fast which uses 32. Could you comment on the impact of this design choice? Was this modification empirically validated (e.g., through ablation studies)? Thank you!

**Comment by hungchiayu1** [comment]
Hi, thanks for your question! Is it possible to clarify what do you mean unusual output? Do you mean that the generated output shape is wrong? Or the generated action causes the widowx to crash into the environment (from what i see in the picture)

We did not retrain or modify the fast tokenizer in bridge  or fractal datasets.

We set the action_dim to 7 because the action space dimension of our training dataset(Open X embodiment) is 7. For pi0-fast, I think they trained on their own dataset that has action space of up 32 dim.

**Comment by DelinQu** [comment]
The output action is right, but inaccurate. The robot arm follows the instructions for the eggplate toy but rarely grasps it successfully. The robotic arm twists/deforms when placing the eggplant back into the basket.

**Comment by lucasjinreal** [comment]
Nora works great, both on libero and on my own dataset. I haven't seen any issue.
